packages/components/app/styles/components/dropdown.scss

```diff
@@ -110,36 +111,41 @@ $hds-dropdown-list-item-border-radius: 5px;
   }
 }
 
-
-// MORE
+// TOGGLE/MORE
 
 // these values later may come from the design tokens
 $size-props: (
-  "small": (
-    "size": 28px
+  'small': (
```

Comment:
[question] in all the other components we're using double quotes in Scss, why these changes in this file?

Reply:
It's probably stylelint. I can undo it (I did think single quotes were correct for CSS though)

Reply:
they're both correct, only most of the files are using double quotes